python/py-pytorch/Portfile

```diff
@@ -109,6 +140,11 @@ if {${name} ne ${subport}} {
         }
     }
 
+    test.run        yes
+    test.cmd        py.test-${python.branch}
+    test.target     test/test_torch.py
+    test.env        PYTHONPATH=${worksrcpath}/build/lib
+
```

Comment:
I am pretty sure this port compiles stuff, so likely the PYTHONPATH isn't correct here either (see above). The tests should work without having the port installed; please verify that this is indeed the case.

Reply:
Thanks done.

Reply:
Note that all the PT tests need a GPU, so unless and until Apple stops fighting with Nvidia, these tests are deprecated. I stuck the tests that will work into an `if` block if it’s ever possible to plug in a Tesla.